pkg/actions/cluster/delete.go

```diff
@@ -0,0 +1,115 @@
+package cluster
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/weaveworks/eksctl/pkg/cfn/manager"
+	"github.com/weaveworks/eksctl/pkg/fargate"
+
+	api "github.com/weaveworks/eksctl/pkg/apis/eksctl.io/v1alpha5"
+	"github.com/weaveworks/eksctl/pkg/eks"
+	"github.com/weaveworks/eksctl/pkg/elb"
+	"github.com/weaveworks/eksctl/pkg/kubernetes"
+	ssh "github.com/weaveworks/eksctl/pkg/ssh/client"
+	"github.com/weaveworks/eksctl/pkg/utils/kubeconfig"
+
+	"github.com/kris-nova/logger"
+)
+
+func deleteSharedResources(cfg *api.ClusterConfig, ctl *eks.ClusterProvider, clientSet kubernetes.Interface, waitTimeout time.Duration) error {
+	clusterOperable, _ := ctl.CanOperate(cfg)
```

Comment:
why do we not care about the error here?

Reply:
this was copied from elsewhere, but looking at the function it states `// CanOperate return true when a cluster can be operated, otherwise it returns false along with an error explaining the reason`. I'm guessing that if we error in finding out if a cluster is operable, its not worth exiting, your better off just assuming its not operable and continuing. A debug log of the error stating that is probably worth while, I'll add one.